distribution/pom.xml

```diff
@@ -83,6 +74,7 @@
 				<executions>
 					<execution>
 						<id>attach-javadoc</id>
+						<phase>install</phase>
```

Comment:
And it will consistently... not work (except for the release profile, which is untouched). I will take a brief look to see if there is any way to force the attach to happen before the install plugin ; otherwise I'll go with the profile option.   If only the javadoc plugin was incremental.